modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v3/form_data.rb

```diff
@@ -0,0 +1,247 @@
+
+            BENEFIT_TYPE_CODES[benefit_type]
+          end
+
+          def informal_conference
+            higher_level_review.informal_conference ? 1 : 'Off'
+          end
+
+          # rubocop:disable Naming/VariableNumber
+          def conference_8_to_12
+            informal_conference_time('veteran', '800-1200 ET')
+          end
+
+          def conference_12_to_1630
+            informal_conference_time('veteran', '1200-1630 ET')
+          end
+
+          def conference_rep_8_to_12
+            informal_conference_time('representative', '800-1200 ET')
+          end
+
+          def conference_rep_12_to_1630
+            informal_conference_time('representative', '1200-1630 ET')
+          end
+          # rubocop:enable Naming/VariableNumber
+
+          def rep_first_name
+            higher_level_review.informal_conference_rep&.dig('firstName')
+          end
+
+          def rep_last_name
+            higher_level_review.informal_conference_rep&.dig('lastName')
+          end
+
+          def rep_phone_area_code
+            return unless rep_domestic_phone?
+
+            higher_level_review.informal_conference_rep_phone.area_code
+          end
+
+          def rep_phone_prefix
+            return unless rep_domestic_phone?
+
+            higher_level_review.informal_conference_rep_phone.phone_number&.first(3)
+          end
+
+          def rep_phone_line_number
+            return unless rep_domestic_phone?
+
+            higher_level_review.informal_conference_rep_phone.phone_number&.last(4)
+          end
+
+          def rep_phone_extension
+            # Unlike with the veteran's phone fields, there is enough space to the right of the domestic claimant phone
+            # fields that we can print the extension next to them instead of filling the international number field as
+            # a fallback.
+            return unless rep_domestic_phone?
+
+            ext = higher_level_review.informal_conference_rep_phone.phone_number_ext
+            "x#{ext}" if ext.present?
+          end
+
+          def rep_international_phone
+            higher_level_review.informal_conference_rep_phone.to_s unless rep_domestic_phone?
+          end
+
+          def rep_email
+            higher_level_review.informal_conference_rep&.dig('email')
+          end
+
+          def veteran_claimant_signature
+            "#{signing_appellant.full_name[0...MAX_SIGNATURE_LENGTH]}\n- Signed by digital authentication to api.va.gov"
+          end
+
+          def veteran_claimant_date_signed_month
+            appellant_local_time.strftime '%m'
+          end
+
+          def veteran_claimant_date_signed_day
+            appellant_local_time.strftime '%d'
+          end
+
+          def veteran_claimant_date_signed_year
+            appellant_local_time.strftime '%Y'
+          end
+
+          # NOTE: Rep signature is not yet supported - only veteran/claimant
```

Comment:
I omitted the rep signature fields because of [this comment](https://github.com/department-of-veterans-affairs/vets-api/blob/f371b420f03eeba7ecfc9bd46b6a8f894e8499f5/modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v2/structure.rb#L91) on v2 of the HLR form